codes/ruby/chapter_backtracking/preorder_traversal_iii_template.rb

```diff
@@ -0,0 +1,69 @@
+=begin
+File: preorder_traversal_iii_template.rb
+Created Time: 2024-04-07
+Author: hoangtien2k3 (hoangtien2k3qx1@163.com)
+=end
+
+require_relative 'utils'
+
+# 判断当前状态是否为解
+def is_solution(state)
```

Comment:
Please add a question mark after every boolean method

```suggestion
def is_solution?(state)
```